src/main/java/frc/robot/commands/Zeroing/ZeroingAlgaeintake.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.Zeroing;
+
+import edu.wpi.first.units.AngularVelocityUnit;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.TimeUnit;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.units.measure.Time;
+import edu.wpi.first.units.measure.Velocity;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants.constElevator;
+import frc.robot.subsystems.Elevator;
+
+public class ZeroingAlgaeintake extends Command {
+  Elevator subElevator;
+
+  boolean zeroingSuccess = false;
+  Measure<TimeUnit> zeroingTimestamp = Units.Seconds.of(0);
+
+  Measure<AngularVelocityUnit> lastRotorVelocity = Units.RotationsPerSecond.of(0);
+
+  public void ManualZeroElevator(Elevator subElevator) {
+    this.subElevator = subElevator;
+
+    addRequirements(subElevator);
+  }
```

Comment:
Is the command zeroing algae or zeroing elevator?

This is named zero algae but has zero elvator code